telluric/georaster.py

```diff
@@ -608,7 +608,7 @@ def __init__(self, image=None, affine=None, crs=None,
         """
         super().__init__(image=image, band_names=band_names, shape=shape, nodata=nodata)
         self._affine = deepcopy(affine)
-        self._crs = CRS(copy(crs)) if crs else None  # type: Union[None, CRS]
+        self._crs = None if crs is None else CRS(crs)  # type: Union[None, CRS]
```

Comment:
How is this line different to what it was done before? (Just curious, I don't immediately see a difference)

Reply:
It used to evaluate trueness of the input crs and apparently an empty crs evaluates to False (`bool(CRS())==False`), so I changed it to explicitly check if input is `None` so it accepts an empty crs. 

Also, I removed the `copy()` because it raised an exception with an empty `CRS`... but since it creates a new instance based on the input, they are different objects and have different `id`, which I understand was the goal of copying it.

Reply:
Fair enough, thanks for the explanation!